steps/image-build/MAINTAINERS

```diff
@@ -5,5 +5,5 @@
 # This file roughly follows the format of an OWNERS file
 # (https://github.com/bkeepers/OWNERS).
 
-# Nebula Community
-@puppetlabs/nebula-community
+# Relay Community
+@puppetlabs/relay-community
```

Comment:
I think we can actually remove this file, at least until we figure out what we want to do with prow.